### ISSUE TITLE
error when trying to run displaycal on kde neon

### ISSUE THREAD

**Issue by cogsfx** [eoyilmaz/displaycal-py3#382]
**Describe the bug**

**To Reproduce**
I just run displaycal in the terminal and then I get this error

```shell
~/D/displaycal-py3 (develop)> displaycal                                                                          (base) 
Acquired lock file: <DisplayCAL.main.AppLock object at 0x79beb6dfdca0>
displaycal 3.9.12 2024-06-26T09:23:33Z
neon 22.04 jammy x86_64
Python 3.12.3 | packaged by Anaconda, Inc. | (main, May  6 2024, 19:46:43) [GCC 11.2.0]
Faulthandler 
wxPython 4.2.1 gtk3 (phoenix) wxWidgets 3.2.2.1
Encoding: utf-8
File system encoding: utf-8
listening
writing to lock file: port: 15411
/home/alex/.venv/lib/python3.12/site-packages/DisplayCAL/colord.py:40: Warning: DBusObjectError('org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.ColorManager was not provided by any .service files: org.freedesktop.ColorManager')
  warnings.warn(safe_str(exception), Warning)
NoneType: None
┌──────────────────────────────────────────────────────────────────────────────┐
│ NoneType: None                                                               │
└──────────────────────────────────────────────────────────────────────────────┘
Traceback (most recent call last):
  File "/home/alex/.venv/lib/python3.12/site-packages/DisplayCAL/main.py", line 550, in main
    _main(module, name, app_lock_file_name)
  File "/home/alex/.venv/lib/python3.12/site-packages/DisplayCAL/main.py", line 486, in _main
    raise ResourceError(
DisplayCAL.debughelpers.ResourceError: resources.notfound.error
lang/en.yaml
┌──────────────────────────────────────────────────────────────────────────────┐
│ Traceback (most recent call last):                                           │
│   File "/home/alex/.venv/lib/python3.12/site-packages/DisplayCAL/main.py",   │
│ line 550, in main                                                            │
│     _main(module, name, app_lock_file_name)                                  │
│   File "/home/alex/.venv/lib/python3.12/site-packages/DisplayCAL/main.py",   │
│ line 486, in _main                                                           │
│     raise ResourceError(                                                     │
│ DisplayCAL.debughelpers.ResourceError: resources.notfound.error              │
│ lang/en.yaml                                                                 │
└──────────────────────────────────────────────────────────────────────────────┘
Exiting displaycal
Ran application exit handlers
fish: Job 1, 'displaycal' terminated by signal SIGSEGV (Address boundary error)
```

**Expected behavior**

I am expecting the ui to launch

**Screenshots**
n/a as I copied above the error I am getting

**Versions (please complete the following information):**
 - OS: KDE Neon
 - Python Version 3.11
 - ArgyllCMS Version [e.g. 3.0.2, 3.1.0, 3.2.0]
 - DisplayCAL Version 3.9.12

**Additional context**
I followed the manual python venv installation which worked without any error, but then I can't launch displaycal due to the error given

**Comment by jwwilliams1994** [comment]
I fixed this myself by copying everything from the displaycal-py3/DisplayCAL folder to displaycal-py3/.venv/share/DisplayCAL. During build/install, the necessary assets aren't copied over I guess.

**Comment by eoyilmaz** [comment]
This should now be fixed in the development branch and will be released with DisplayCAL 3.9.14.
